### ISSUE TITLE
Getting error when trying to wrap a model with a tf keras Normalization layer

### ISSUE THREAD

**Issue by tk3016** [fabiodimarco/tf-levenberg-marquardt#11]
Hello,
Thanks for building the capability to use LM algorithm with Tensorflow.
When I am trying to use my own dataset to build a shallow neural net using Tensorflow 2.9.0 for which I need to normalize the features using tf.keras.layers.Normalization(axis=-1), I end up getting an error when trying to wrap the model as instructed for using the LM algorithm. The error description is as follows:
_All 'axis' values to be kept must have known shape. Got axis: (-1,), input shape: [None, None], with unknown axis at index: 1_

Many thanks for all your help in advance.

Best wishes,
Tanuj

**Comment by fabiodimarco** [comment]
Hi Tanuj,

could you provide me a code to replicate the issue? Instead of using the real database you can also use a randomly generated one with same input/output dimensions.

**Comment by tk3016** [comment]
Hi 

Thanks for your prompt response. 
I am attaching a different code with the same issue I am facing which is based on the regression tutorial on Tensorflow's website.
[tf_regression_tutorial_with_error.zip](https://github.com/fabiodimarco/tf-levenberg-marquardt/files/8764675/tf_regression_tutorial_with_error.zip)

**Comment by fabiodimarco** [comment]
Hi Tanuj,

I ran your notebook, the error is not due to the ModelWrapper, but it is caused by this line of code:
`tf.keras.models.clone_model(dnn_model_shallow)`

Actually you do not need to make a clone of the model, you can just pass it directly to the ModelWrapper.

```
model_wrapper = lm.ModelWrapper(dnn_model_shallow)

model_wrapper.compile(
    optimizer=tf.keras.optimizers.SGD(learning_rate=0.1),
    loss=lm.MeanSquaredError())

history = model_wrapper.fit(
    train_features,
    train_labels,
    validation_split=0.2,
    verbose=1,
    epochs=100
)
```
Let me know if this solves your problem.

Best,
Fabio
